CMakeLists.txt

```diff
@@ -91,3 +85,9 @@ endif(USE_CUSTOM_CLOCK)
 
 # Enable sending binaries to robots
 include(RobotSend)
```

Comment:
You know, I think you can delete this and the corresponding file now. We don't use this to send data to robots, we use `./b install` so this just vestigial code